builder/Makefile

```diff
@@ -1,7 +1,6 @@
 ORG=libhkl
-REPO=sandbox
+REPO=phoenix
 LOGFILE=build-log.txt
-NET=host-bridge
```

Comment:
Will now need to keep the definition of NET in this file.

Reply:
done